docs/Consensus/Consensus.md

```diff
@@ -5,7 +5,7 @@ With no need for POW/POS in a permissioned network, Quorum instead offers multip
 * __Raft-based Consensus__: A consensus model for faster blocktimes, transaction finality, and on-demand block creation.  See [Raft-based consensus for Ethereum/Quorum](../raft) for more information 
 
 
-* __Istanbul BFT (Byzantine Fault Tolerance) Consensus__: A PBFT-inspired consensus algorithm with transaction finality, by AMIS.  See [Istanbul BFT Consensus documentation](https://github.com/ethereum/EIPs/issues/650), the [RPC API](../ibft/istanbul-rpc-api.md), and this [technical web article](https://medium.com/getamis/istanbul-bft-ibft-c2758b7fe6ff) for more information
+* __Istanbul BFT (Byzantine Fault Tolerance) Consensus__: A PBFT-inspired consensus algorithm with transaction finality, by AMIS.  See [Istanbul BFT Consensus documentation](../ibft/ibft), the [RPC API](../ibft/istanbul-rpc-api), and this [technical web article](https://medium.com/getamis/istanbul-bft-ibft-c2758b7fe6ff) for more information
```

Comment:
should we say "immediate transaction finality", as other algorithms do have some notion of finality too?

Reply:
Updated the documentation to reflect this